.github/workflows/test.yml

```diff
@@ -29,6 +29,8 @@ jobs:
       uses: dtolnay/rust-toolchain@nightly
     - name: Install Wasm toolchain
       run: rustup target add wasm32-unknown-unknown
+    - name: Install Zepter
+      run: cargo install zepter --version 0.14.0 --locked -q -f --no-default-features && zepter --version
```

Comment:
doesn't need to happen in this PR but at some stage I want to use https://github.com/taiki-e/install-action (powered by https://github.com/cargo-bins/cargo-binstall) to install tools. I don't know if zepter is compatible with it but we could try and see

Reply:
Yea i wanted to use `cargo-install` as well. I think it just pulls binaries from the the GH release page. Will explore it and then open another MR when its done.